data/saas/dataset/rollbar_dataset.yml

```diff
@@ -0,0 +1,82 @@
+dataset:
+- fides_key: rollbar_connector
+  name: Rollbar Dataset
+  description: A sample dataset representing the Rollbar connector for Fidesops
+  collections:
+  - name: projects
+    fields:
+    - name: id
+      data_categories: [system.operations]
+      fidesops_meta:
+        data_type: integer
+        primary_key: true
+    - name: account_id
+      data_categories: [system.operations]
+      fidesops_meta:
+        data_type: integer
+    - name: name
+      data_categories: [system.operations]
+      fidesops_meta:
+        data_type: string
+
+  - name: project_access_token
+    fields:
+      - name: access_token
+        data_categories: [ system.operations ]
```

Comment:
Minor: remove the spacing to be consistent with other occurrences of `[system.operations]`